src/hoard.rs

```diff
@@ -44,7 +44,12 @@ impl Hoard {
         self
     }
 
-    pub fn start(&mut self) -> (String, bool) {
+    pub async fn start(&mut self) -> (String, bool) {
+        if !compare_with_latest_version().await {
+            println!(
+                "Latest Version Available at https://github.com/Hyde46/hoard \nPlease update."
```

Comment:
I think I'd prefer something along the lines of 
"Running hoard<version> a newer version <new_version> is available \n Download at <link>"
So that the user knows whether it's a minor patch or a big version bump. This might change their willingness to update if the difference is bigger